common.py

```diff
@@ -0,0 +1,110 @@
+import csv
+import os
+import urllib.request
+
+from IPython import embed
+
+
+class FileLoaderMixin():
+
+    def load_file(self, file_path):
+
+        # make sure that we only try to load supported file types
+        supported_file_types = {'.csv', '.txt'}
+        current_file_type = file_path[file_path.rfind('.'):]
+        if not current_file_type in supported_file_types:
+            raise ValueError("The FileLoaderMixin currently supports {supported_file_types} but not "
+                             "{current_file_type}.")
```

Comment:
We should see if this will be lab policy, but I don't like bare string concat -- preferring adding a "+" sign, since they'll need it in Javascript and other languages.

Reply:
Agreed and fixed.